### ISSUE TITLE
Shasum Check Fail when installing over npm

### ISSUE THREAD

**Issue by LA1CH3** [mikaelbr/node-notifier#83]
Running node 4.0.0 and npm 2.14.2 I get the following error when trying to install:
```
npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "node-notifier"
npm ERR! node v4.0.0
npm ERR! npm  v2.14.2

npm ERR! shasum check failed for /tmp/npm-34171-374996e4/registry.npmjs.org/node-notifier/-/node-notifier-4.3.1.tgz
npm ERR! Expected: 94a3e35c388938b4d9fa196fed6b36e9ab671119
npm ERR! Actual:   95370ca4ed0e0360d5a7869967759a3df2e769f3
npm ERR! From:     http://registry.npmjs.org/node-notifier/-/node-notifier-4.3.1.tgz
npm ERR! 
```
I have run "npm cache clean" and deleted my tmp folder but the error persists on multiple re-installs.

**Comment by kurisubrooks** [comment]
I'm running Node 4.1.2 and the latest npm and haven't come across this problem. 
I'll leave this as unconfirmed for now, and will wait to see what @mikaelbr has to say.

**Comment by mikaelbr** [comment]
That's weird. Is it just an issue with node-notifier, or other packages as well? But I figure its a npm/node error as it's not the same for everyone it can't be a corrupt tar-ball, right?

**Comment by LA1CH3** [comment]
The issue originally stemmed from `gulp-notify` however as of this morning we have resolved our issues by using 2.2.0 of `gulp-notify`, which uses `node-notifier` at 4.1.0. We must have been explicitly calling an old version of `gulp-notify` in our package previously - anyways, I'm closing. Thanks for looking into though.

**Comment by mikaelbr** [comment]
Glad it worked out!
